deployments/workloads/slight/workload.yaml

```diff
@@ -15,8 +15,22 @@ spec:
       runtimeClassName: wasmtime-slight
       containers:
         - name: testwasm
-          image: docker.io/mossaka/slight-try:latest
+          image: docker.io/mossaka/slight-try3:awsdynamodb
```

Comment:
We want to have an image reference similar to: https://github.com/deislabs/containerd-wasm-shims/blob/cb07e66d03625918cd37c22b93c6b760133f609e/deployments/workloads/spin/workload.yaml#L18.